packages/sdks/src/context/builder.context.lite.ts

```diff
@@ -7,6 +7,7 @@ export default createContext<BuilderContextInterface>({
   state: {},
   setState: () => {},
   apiKey: null,
+  apiVersion: 'v2',
```

Comment:
This can actually be `undefined` too. Since `getContent` accepts undefined values, it's better to only have one default there instead of another one here.

```suggestion
  apiVersion: undefined,
```